code/images/SCIENCE_files/style.css

```diff
@@ -0,0 +1,438 @@
+body{
+  margin:0px;
+  padding:0px;
+  font-family: 'The Nautigal';
```

Comment:
Nice font style! However I've heard it's good to always have in mind to write out two different font families. Like:

font-family: 'The Nautigal', 'sans serif', just to make sure the browser can show it. If the browser doesnt have access to the font style Nautigal it will choose sans serif! :)

Reply:
Yup I will add that it is good to have option if browser does-not accept the fonts